libraries/ms-common/src/stm32f0xx/interrupt_mcu.c

```diff
@@ -0,0 +1,92 @@
+#include "interrupt_mcu.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "status.h"
+#include "stm32f0xx_exti.h"
+#include "stm32f0xx_misc.h"
+#include "stm32f0xx_rcc.h"
+#include "stm32f0xx_syscfg.h"
+
+// Channels are NVIC channels which handle prioritization of interupts.
+#define NUM_INTERRUPT_MCU_CHANNELS 32
+// Lines are external interrupt channels often grouped to a single
+#define NUM_INTERRUPT_MCU_LINES 32
+
+static InterruptPriority s_interrupt_mcu_priorities[NUM_INTERRUPT_MCU_CHANNELS];
+static bool s_interrupt_mcu_initialized = false;
+
+void interrupt_mcu_init(void) {
+  if (!s_interrupt_mcu_initialized) {
```

Comment:
Do you think preventing re-initialization is necessary? I think it's useful for testing.